src/main/resources/db/migration/V2__update_role_table.sql

```diff
@@ -0,0 +1,74 @@
+ALTER TABLE roles
+ADD `description` varchar(5000),
+ADD responsibilities varchar(5000),
+ADD  job_link varchar(5000);
```

Comment:
wow 5000?

Reply:
lol my google search said job descriptions are around 4000-5000 characters so I went with the larger number. I can make it smaller though! And I guess the job link should be smaller than the description anyway

Reply:
if you think that's what you need. no issues 😆

Reply:
changed these lol!